components/core-listeners/src/main/java/org/eclipse/dirigible/components/listeners/synchronizer/ListenerSynchronizer.java

```diff
@@ -58,7 +58,6 @@ public boolean isAccepted(String type) {
     public List<Listener> load(String location, byte[] content) {
         Listener listener = GsonHelper.GSON.fromJson(new String(content, StandardCharsets.UTF_8), Listener.class);
         listener.setLocation(location);
-        listener.setName("");
```

Comment:
use setName() with something similar to:
FilenameUtils.getBaseName(file.getFileName().toString())
where 'file' is a Path from 'location'